Apps/Contoso.WPF.Demo.DotNetCore/Settings.settings

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version='1.0' encoding='utf-8'?>
-<SettingsFile xmlns="http://schemas.microsoft.com/VisualStudio/2004/01/settings" CurrentProfile="(Default)" GeneratedClassNamespace="Contoso.WPF.Demo.Properties" GeneratedClassName="Settings">
+<SettingsFile xmlns="http://schemas.microsoft.com/VisualStudio/2004/01/settings" CurrentProfile="(Default)" GeneratedClassNamespace="Contoso.WPF.Demo.DotNetCore" GeneratedClassName="Settings">
```

Comment:
Did you test that? Shouldn't it be Properties.DotNetCore?

Reply:
VS did this change and i guess, this is correct change as namespace is Contoso.WPF.Demo.DotNetCore